doc/applications.md

```diff
@@ -0,0 +1,151 @@
+# Creating applications
+
+RxJS is a great tool to keep your code less error prone. It does that by using pure and stateless functions. But applications are stateful, so how do we bridge the stateless world of RxJS with the stateful world of our applications?
+
+### Vanilla JS
+```js
+const button = document.querySelector('button');
+Rx.Observable.fromEvent(button, 'click')
+  .scan(count => count + 1, 0)
+  .subscribe(count => document.querySelector('#count').innerHTML = count);
+```
```

Comment:
This example could at least have a textual explanation: "detects clicks... accumulates state starting at 0 and incrementing every time a click happens... sets the innerHTML to be the state..."